package.json

```diff
@@ -152,5 +154,8 @@
     "minimist": ">=1.2.6",
     "node-forge": ">=1.3.0",
     "underscore": "1.13.2"
-  }
-}
+  },
+  "dependencies": {
+    "@bbc/apache2-license-checker": "^1.1.4"
```

Comment:
@micoferdinand98 We'll need this in the future for a different task but not for the one you are currently working on.
Here the point is to check whether our own packages declare their license as Apache 2 or not (and fail if not). The license of the depdencies of said packages is not in scope here to be checked for.

Reply:
I have also removed this one since its not currently being needed at the task at hand and not also in scope to be checked.